unity-client/Assets/Scripts/MainScripts/DCL/Helpers/NFTHelper/NFTMarkets/OpenSea_Internal/Request.cs

```diff
@@ -0,0 +1,196 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking;
+
+#pragma warning disable CS0162
+
+namespace DCL.Helpers.NFT.Markets.OpenSea_Internal
+{
+    internal class RequestController
```

Comment:
Maybe we can rename this to `OpenSeaRequestController`? As i feel the name seems too generic for this class purpose and could cause some confusions. Same with the rest of the class names. Yes, they are enclosed by a custom namespace, but namespace scope is too lenient because of `using`.